src/components/FilterPanel/FilterPanel.styles.ts

```diff
@@ -0,0 +1,81 @@
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import { styled } from '@mui/material/styles';
+
+import { FILTER_BUTTON_SIZES } from './FilterPanel.constants';
+
+import type { FilterVariant } from './FilterPanel.types';
+import type { Theme } from '@mui/material/styles';
+
+interface FilterPanelContainerProps {
+  panelVariant: 'primary' | 'secondary';
+  size?: 'medium' | 'large' | 'xlarge';
+}
+
+export const FilterPanelContainer = styled(Box, {
+  shouldForwardProp: (prop) => prop !== 'panelVariant',
+})<FilterPanelContainerProps>(({ theme, panelVariant }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  border: `1px dashed ${theme.palette.divider}`,
+  borderRadius: theme.shape.borderRadius,
+  padding: theme.spacing(1),
+  gap: 0,
+  maxWidth: '300px',
+  backgroundColor:
+    panelVariant === 'secondary' ? theme.palette.grey[900] : theme.palette.background.paper,
+}));
+
+export const FilterButton = styled(Button, {
+  shouldForwardProp: (prop) => prop !== 'selected' && prop !== 'panelVariant',
+})<{
+  selected?: boolean;
+  panelVariant?: FilterVariant;
+}>(({
+  theme,
+  selected,
+  panelVariant,
+}: {
+  theme: Theme;
+  selected?: boolean;
+  panelVariant?: FilterVariant;
+}) => {
+  const isPrimary = panelVariant === 'primary' || !panelVariant;
+  const size = FILTER_BUTTON_SIZES.xlarge;
+
+  const secondaryFill =
+    theme.palette.mode === 'light' ? theme.palette.secondary.light : theme.palette.secondary.light;
+
+  return {
+    justifyContent: 'center',
+    minHeight: size.height,
+    maxWidth: '300px',
```

Comment:
Again, maxWidth is hardcoded as '300px' here. This violates the project's 'no px hardcoded' rule and duplicates the container maxWidth.

**Sugerencia:**
```typescript
Use theme.spacing or inherit the container width. Example:

maxWidth: '100%',
// or use theme.spacing(...) as in container
```